### PR TITLE
chore: DLT-1692 vue examples blog post

### PR DESCRIPTION
**Pull request by juliodialpad**
# Vue examples blog post

## Obligatory GIF (super important!)

Obligatory GIF

## :hammer_and_wrench: Type Of Change

These types will not increment the version number, but will still deploy to documentation site on release:

- [x] Other (chore)

## :book: Jira Ticket

https://dialpad.atlassian.net/browse/DLT-1692

## :book: Description

Added blog post to announce we now have Vue examples on https://dialtone.dialpad.com

## :bulb: Context

We finished most of the tasks on https://dialpad.atlassian.net/browse/DLT-221 epic long ago, but we haven't announced it properly yet.

## :pencil: Checklist

For all PRs:

- [x] I have ensured no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
- [x] I have reviewed my changes.

## :crystal_ball: Next Steps

- Should I include the announcement of the prototype on https://dialtone.dialpad.com/vue/index.html?path=/story/prototypes-dialpad--default ?

### CODE REVIEW

**Review by ninamarina — APPROVED** [review]
Thanks!

File: apps/dialtone-documentation/docs/about/whats-new/posts/2024-8-13.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+# Hi Dialers 💜
+
+We have a pretty short announcement, but packed with great news! 🥳
+
+## Dialtone documentation
+
+We are currently on an effort to unify our two documentation sites [Dialtone vue - Storybook](https://dialtone.dialpad.com/vue) and [Dialtone](https://dialtone.dialpad.com) into [Dialtone](https://dialtone.dialpad.com) only.
+
+Today, We are happy to announce the availability of **Vue component examples** on the documentation site. That way is easier for you to understand the usage of it, slots, props, events, see some examples and copy the code directly so you can start using our components right away!
```
Comment (iropolo) [inline apps/dialtone-documentation/docs/about/whats-new/posts/2024-8-13.md]: `Today, We` -> `Today, we`

File: apps/dialtone-documentation/docs/about/whats-new/posts/2024-8-13.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+We are continuosly working on making Dialtone better, so your days working with Dialtone are more enjoyable and hassle free.
```
Comment (iropolo) [inline apps/dialtone-documentation/docs/about/whats-new/posts/2024-8-13.md]: `continuosly` -> `continuously`

**Review by iropolo — APPROVED** [review]

File: apps/dialtone-documentation/docs/about/whats-new/posts/2024-8-13.md (hunk 1 above)
Comment (iropolo) [inline apps/dialtone-documentation/docs/about/whats-new/posts/2024-8-13.md]: `That way is easier for you to understand the usage` -> `This makes it easier for you to understand the usage`

**Review by braddialpad — APPROVED** [review]
Perfect, thanks!

**Pull request merged**